pkg/validate/dynamic/platformworkloadidentityprofile.go

```diff
@@ -103,7 +87,7 @@ func (dv *dynamic) validateClusterMSIPermissions(ctx context.Context, oid string
 			return err
 		}
 
-		err = dv.validateActionsByOID(ctx, &pid, actions, &oid)
+		err = dv.validateActionsByOID(ctx, &pid, actions, nil)
```

Comment:
We should change the comment for validateActionsByOID.
https://github.com/Azure/ARO-RP/blob/6c56ef7d153972d00a72a7f93afedfc74e0554c1/pkg/validate/dynamic/dynamic.go#L431
Because oid is nil though it's supposed to be called for MIWI cluster.

Reply:
Yes will change it, it is still need to be non-nil for MIWI cluster though, only Cluster MSI in MIWI cluster will have oid as nil.